owon_psu/__init__.py

```diff
@@ -36,10 +36,15 @@ def _cmd(self, command, accept_silent=False, timeout=None):
       raise Exception("Connection is not open!")
     self.ser.write(bytes(command, 'utf-8') + b"\n")
     self.ser.timeout = timeout if timeout is not None else self.timeout
-    ret = self.ser.readline().decode('utf-8')
-    if not ret.endswith("\r\n") and not accept_silent:
+    ret = self.ser.readline().decode('utf-8', errors='replace')
+    if not ret and not accept_silent:
       raise Exception(f"No response for command: '{command}'!")
-    return ret[:-2]
+
+    # Some firmware revisions terminate responses with '\r' only or '\n' only.
+    ret = ret.rstrip("\r\n")
+    if not ret and not accept_silent:
+      raise Exception(f"No response for command: '{command}'!")
```

Comment:
should be no reason to double up this error handling. let's just clean up the terminators before doing the check and return